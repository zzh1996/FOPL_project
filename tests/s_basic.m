Begin
	Var x End
	Function test Paras i Begin
		If Gt i 10 Begin
			Return Plus i Minus i 1 
			End
		Else Begin 
			If Gt i 5 Begin
				Return Mult i Div i 2
				End
			Else Begin
				Return Mod i 2
				End
		End	
	End
	Read x
	While Gt x 0 Begin
		Print Apply test Argus x End
		Assign x Minus x 1
	End
End