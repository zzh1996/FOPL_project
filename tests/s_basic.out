23 21 50 36 32 21 18 1 0 1 0 1
